RibonDAO/app
# Feat/random post donation color

# Description

Adds random color to the post donation screen

## Type of change

Please delete options that are not relevant.


## How to test

Make some donations to see the new random colors

<img src="https://github.com/user-attachments/assets/b11215a7-f8b1-491c-bf21-7ae100668383" width="200">
<img src="https://github.com/user-attachments/assets/a1aa0217-05ad-44a0-b992-c804d1d870bb" width="200">
